sylt-compiler/src/typechecker.rs

```diff
@@ -1383,7 +1397,7 @@ impl TypeChecker {
                     .collect(),
                 Box::new(self.inner_bake_type(ret, seen)),
             ),
-            Type::Blob(name, fields) => RuntimeType::Blob(
+            Type::Newblob(name, fields, _) | Type::Blob(name, fields) => RuntimeType::Blob(
```

Comment:
This is used for type-errors, so it's a good idea to add a new field to `RuntimeType` and split this, it will make errors clearer. (`RuntimeType` is in sylt-common/src/ty.rs and there called `Type`)